docs/lang/articles/contribution/write_test.md

```diff
@@ -13,6 +13,16 @@ Normally we write functional tests in Python.
 For example, you've just added a utility function `ti.log10`. Now you
 want to write a **test** to ensure that it functions properly.
 
+:::note
+
+Before running the test set by `pytest`, you need to install the corresponding
```

Comment:
We should use the launcher named `tests/run_tests.py` , directly calling `pytest` will fail.

Reply:
Thank you for the suggestion. I modified the command in the latest commit.